app/src/main/java/com/github/onlynotesswent/ui/user/EditProfile.kt

```diff
@@ -157,7 +185,7 @@ fun EditProfileScreen(
                                       "Error while updating user: ${exception.message}",
```

Comment:
I don't think the Toast displayed to the user should show the exception that it failed with, as it is a technical detail and not user-friendly. 
Instead, we could show a message like `"Unable to update user information. Please try again later."` or `"Oops! Something went wrong. Please try again later."`
If it fails because username is taken it could be `"Username is already taken. Please choose a different one."`

Reply:
Yes it's implemented now